### DISCUSSION TITLE
Fastestvpn add request

### DISCUSSION THREAD

**Discussion by bijfy** [qdm12/gluetun#376] [Ideas]
Hi can you please add fastestvpn.com  for gluten vpn docker https://support.fastestvpn.com/download/openvpn-tcp-udp-config-files/
Here are the config files in case you need them.  Thanks

**Comment by qdm12** [comment]
Should be done this weekend 👍  I'll ping you here so you can test a beta image for fastestvpn. Thanks for the suggestion.

**Reply by qdm12** [reply]
@bijfy Can you try with the image `qmcgaw/gluetun:fastestvpn`? The wiki should have the information necessary to get started: https://github.com/qdm12/gluetun/wiki/Environment-variables Let me know if it works so that we can merge in the `:latest` and future releases 😉

**Reply by bijfy** [reply]
Thanks .will.try this weekend .

**Reply by qdm12** [reply]
Hey there, just a reminder to try it out 😉 Thanks!

**Comment by bijfy** [comment]
Sure. I will

**Comment by bijfy** [comment]
docker: Error response from daemon: Get https://registry-1.docker.io/v2/: dial tcp: lookup registry-1.docker.io on 192.168.64.1:53: read udp 192.168.64.4:57939->192.168.64.1:53: read: connection refused.
 i tried. but need to figure it out first. any suggestions. thanks

**Reply by qdm12** [reply]
Try `docker pull alpine`. Does it work?

**Reply by qdm12** [reply]
I think your DNS at 192.168.64.1 refuses your query to resolve the Docker registry dns address. I can't really help apart from telling you that.

**Reply by qdm12** [reply]
Did you manage to make it work in the end? I would like to go ahead and merge the support for Fastestvpn.

**Comment by bijfy** [comment]
i am on raspberry pi 4

**Comment by bijfy** [comment]
Sorry . I couldn't.  I will try again once I made my raspberry pi 4 from
scratch.  Thanks for your help .

**Reply by qdm12** [reply]
But can you run any Docker container at all?

**Comment by bijfy** [comment]
Hey bud
Just to update.  Fastestvpn is running great on your image . Thanks for
your help

**Reply by qdm12** [reply]
Great thanks for letting me know!
